.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "client/vendor/wormhole-william"]
 	path = client/vendor/wormhole-william
 	url = git@github.com:LeastAuthority/wormhole-william.git
+[submodule "feedback-api/feedback-api"]
+	path = feedback-api/feedback-api
```

Comment:
Isn't better to put submodule under client/vendor directory to have unified place?

Reply:
The idea I have with the codebase is that all top-level services reside in the project root. If something needs to be built with alongside the service (such as a library, or in this case a wasm module), then we put it in the `<service>/vendor` folder